suite-native/device-manager/src/components/DeviceManagerModal.tsx

```diff
@@ -79,27 +79,30 @@ export const DeviceManagerModal = ({
                     style={applyStyle(deviceManagerModalWrapperStyle, { insets })}
                 >
                     <Box style={applyStyle(deviceSwitchWrapperStyle, { insets })}>
-                        <ScreenHeaderWrapper>
-                            <HStack
-                                justifyContent="space-between"
-                                alignItems="center"
-                                spacing="medium"
-                                flex={1}
-                            >
-                                <Box style={{ flexShrink: 1 }}>
-                                    {deviceState && (
-                                        <DeviceItemContent
-                                            deviceState={deviceState}
-                                            headerTextVariant="titleSmall"
-                                            isCompact={false}
-                                            isPortfolioLabelDisplayed={true}
-                                        />
-                                    )}
-                                </Box>
-                                {customSwitchRightView}
-                            </HStack>
-                        </ScreenHeaderWrapper>
+                        <Pressable onPress={handleClose}>
+                            <ScreenHeaderWrapper>
+                                <HStack
+                                    justifyContent="space-between"
+                                    alignItems="center"
+                                    spacing="medium"
+                                    flex={1}
+                                >
+                                    <Box flexShrink={1}>
```

Comment:
nit: Should not the deviceState conditionally render also the parent Box? If it will be false, then there will be a unnecessary empty Box. Is it expected?
```suggestion
                                    { deviceState & <Box flexShrink={1}>
```

Reply:
f312d14b01